whylog_vim/consts.py

```diff
@@ -0,0 +1,106 @@
+
+
+class WindowTypes():
+    QUERY = 'whylog_query_output.whylog'
+    TEACHER = 'whylog_teacher.whylog'
+    INPUT = 'whylog_input.whylog'
+    MESSAGE = 'whylog_message.whylog'
+    CASE = 'whylog_case.whylog'
+
+
+class GlobalConsts():
+    BUTTONS_HEADER = '--- Buttons:'
+    EMPTY_LINE = ''
+    END_BRACKET = '<<<'
+    MAIN_BUTTONS = '[save] [verify_rule] [return_to_file] [abandon_rule]'
+    MAIN_HEADER = '# You are using whylog teacher.'
+
+
+class ButtonsMetaConsts():
+    CONSTRAINT = 'constraint'
+    CONSTRAINT_GROUP = 'constraint_group'
+    GROUP = 'group_id'
+    LOG_TYPE = 'log_type'
+    PARAM = 'param'
+    PARSER = 'parser_id'
+    PRIMARY_KEY = 'primary_key'
+    BUTTON = 'button'
+    FUNCTION = 'function'
+
+
+class ParserOutputConsts():
+    GROUP_CONVERTER = 'group %s: %s, match: %s'
+    LINE_BUTTONS = '[copy_line] [delete_line]'
```

Comment:
All button names should be separate consts. Consider classes: ConstraintButtons, ParserButtons ..., that have const button names.